src/Utilities/Microsoft.Build.Utilities.csproj

```diff
@@ -209,15 +209,20 @@
     </Compile>
     <!-- Win32 RC Files -->
     <RCResourceFile Include="native.rc" />
+    
```

Comment:
I care (a bit) about the trailing space characters. Bare newlines for paragraph spacing I wholeheartedly approve of.

(I'm cursed by my ongoing "need" to have visible whitespace on, so I see this sort of things.)